controller/konnect/ops/ops.go

```diff
@@ -171,6 +184,14 @@ func Create[
 
 	logOpComplete(ctx, start, CreateOp, e, err)
 
+	// If the error is a type field error or bad request error, then don't propagate
+	// it to the caller.
+	// We cannot recover from this error as this requires user to change object's
+	// manifest. The entity's status is already updated with the error.
+	if ErrorIsSDKErrorTypeField(err) || ErrorIsSDKBadRequestError(err) {
+		err = nil
+	}
+
```

Comment:
This check is in a couple of places, maybe a dedicated helper like `IgnoreUnrecovableErr` and use it like that
```go
    return e, IgnoreUnrecovableErr(err)
```
wdyt?

Reply:
Good call. 62eb3f5e2d1e5a9b8603a03dd58fbd5fcb180eaf should handle that.